migrations/000001_create_authors_table.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS authors (
+    id SERIAL PRIMARY KEY,
```

Comment:
Bigserial would be better

Reply:
Why exactly? Are you sure that range from 1 to 2billion wouldn't be enough for authors table? I'm not sure we have that much of authors